src/index.js

```diff
@@ -1,6 +1,22 @@
 import prettify from './helpers';
 import { log } from './debug';
 
-export default (schema, data, errors) => {
-  prettify(errors).forEach(e => log(e.print(schema, data).join('\n')));
+export default (options = { schema, mode, indent }) => {
+  const {
+    schema,
+    mode = 'print',
+    indent = 2,
+  } = options;
+
+  return (data, errors) => {
+    const customErrorToText = (error) => error.print(schema, data).join('\n');
+    const customErrorToStructure = (error) => error.getError(schema, data);
```

Comment:
Maybe move these two `functions` out of the returned `function`.

Reply:
fixed: ed36098